AdblockPlusSafariTests/AdblockPlusSafariTests.m

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-2015 Eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/&gt.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "AdblockPlusExtras.h"
+#import "AdblockPlus+Extension.h"
+#import "AdblockPlus+Parsing.h"
+
+@interface AdblockPlusSafariTests : XCTestCase
+
+@end
+
+@implementation AdblockPlusSafariTests
+
+- (void)setUp
+{
+  [super setUp];
+  // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown
+{
+  // Put teardown code here. This method is called after the invocation of each test method in the class.
+  [super tearDown];
+}
+
+- (void)performMergeFilterlists:(NSString *)filterlists
+{
```

Comment:
This method is testing parser login if it is able to process files with filter lists and produces valid json.